src/kvstore/comm.h

```diff
@@ -79,8 +79,35 @@ class Comm {
     return pinned_ctx_;
   }
 
+  /**
+   * \brief Sets gradient compression parameters to be able to
+   * perform reduce with compressed gradients
+   */
+  inline void SetCompress(const std::string& compress,
```

Comment:
nit: formatting

Reply:
Not sure I follow.
Are you referring to the formatting of lines 87 and 88? 
Or do you think perform should be indented to the right?

Reply:
I forget... Maybe is was just bad formatting in this tool.